src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function (num) {
+      this.result += num;
+
+      return this;
+    },
+
+    subtract: function (num) {
+      this.result -= num;
+
+      return this;
+    },
+
+    multiply: function (num) {
+      this.result *= num;
+
+      return this;
+    },
+
+    divide: function (num) {
```

Comment:
```suggestion
    divide(num) {
```
we can simplify that